optaplanner-examples/src/test/java/org/optaplanner/examples/nqueens/solver/tracking/NQueensConstructionHeuristicTrackingTest.java

```diff
@@ -37,27 +36,15 @@
 import org.optaplanner.examples.nqueens.domain.NQueens;
 import org.optaplanner.examples.nqueens.persistence.NQueensGenerator;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertNotNull;
```

Comment:
I may be wrong here, but I ran into something similar elsewhere. JUnit 5, in combination with Surefire, wouldjust silently skip some tests.

See https://stackoverflow.com/questions/36970384/surefire-is-not-picking-up-junit-5-tests for more.

Reply:
Assertions are out of scope of this PR although it makes sense to completely drop dependency on JUnit 4. Maybe a simple regex on the import statement would solve that but I really want to avoid scope creep.

JUnit 5 migration tip:
> Note that you may continue to use assertion methods from org.junit.Assert or any other assertion library ...